apps/tenant-management-webapp/src/app/pages/admin/services/pdf/templates/previewEditor/PreviewTemplate.tsx

```diff
@@ -43,17 +46,22 @@ export const PreviewTemplate: FunctionComponent<PreviewTemplateProps> = ({ chann
 
   const dispatch = useDispatch();
 
-  const indicator = useSelector((state: RootState) => {
-    return state?.session?.indicator;
-  });
+  const indicator = useSelector((state: RootState) => state?.session?.indicator, _.isEqual);
```

Comment:
It is not a good idea to use indicators at this level. For example, if we like to make the button in the template disabled when an action takes place. The best way in my mind is to move the button to a separate component. We can add 'indicator' in that separate component to avoid unnecessary refresh. Another very similar issue is that {indicaot.show && <Any loading indicator> } is not suggested. To avoid the issue, the global page indicator in the components/indicator.tsc comes with an useSelect function to detect the state update. I would wish the action loading indicator in the Page has a similar structure here. I think using _isEqual is not unnecessary here.